libs/common-const/src/common.ts

```diff
@@ -64,6 +64,8 @@ export const V_COW_CONTRACT_ADDRESS: Record<SupportedChainId, string | null> = {
   [SupportedChainId.ARBITRUM_ONE]: null, // doesn't exist!
   [SupportedChainId.BASE]: null, // doesn't exist!
   [SupportedChainId.SEPOLIA]: '0x21d06a222bbb94ec1406a0a8ba86b4d761bc9864',
+  [SupportedChainId.AVALANCHE]: 'TODONETOWRK',
```

Comment:
TODO: revert that one new networks merged in develop. This only need to run the PR locally.